.circleci/config.yml

```diff
@@ -126,6 +129,12 @@ workflows:
       - pylint:
           requires:
             - pip-install
+      - shellcheck/check:
+          dir: ./tools
+          external_sources: true
```

Comment:
```suggestion
```
This is probably not needed, since the utils are also checked themselves (and thus not considered an external source)?

Reply:
Shellcheck fails when I remove this, as `SC1091 (info): Not following: ./tools/utils/_functions.sh was not specified as input (see shellcheck -x).`

See: https://app.circleci.com/pipelines/github/digitalfabrik/integreat-compass/77/workflows/35ab449d-2711-43f3-a7c3-c3cfd5276f3c/jobs/250

Will keep it as it is, if you are fine with this.